src/HashingPassword.cpp

```diff
@@ -0,0 +1,8 @@
+#include "HashingPassword.hpp"
+#include<iostream>
+
+std::string HashingPassword::hashPassword(std::string p)
+ {
+     std::string hashedPassword_ = picosha2::hash256_hex_string(p);
+     return hashedPassword_;
```

Comment:
Code duplication, it's the same as AuthenticationProvider::hashPassword.
Which one should be chosen?